### ISSUE TITLE
Element in tab order lacks a role and name.

### ISSUE THREAD

**Issue by LaurenceRLewis** [highcharts/highcharts#22984]
#### Expected behaviour
Implementing a tabindex of 0 on a element with a generic role causes the element to be considered a User Interface Component (UIC). A UIC must have a valid Role and Name.

#### Actual behaviour
Container with tabindex =0 is missing a valid role and name.

#### Suggestion
Move the tabindex up the DOM tree and place it on the region landmark that has an accessible name

##### Existing
```
<div id="serviceusage-international-bar-line-chart-picker" class="able-Spacing able-Spacing--top--spacing1x" data-highcharts-chart="0" style="overflow: hidden;" aria-hidden="false" role="region" aria-label="Chart. Highcharts interactive chart.">
<div id="highcharts-ID" class="highcharts-container " aria-hidden="false" tabindex="0">
```

##### Recommended
```
<div id="serviceusage-international-bar-line-chart-picker" class="able-Spacing able-Spacing--top--spacing1x" data-highcharts-chart="0" style="overflow: hidden;" aria-hidden="false"  tabindex="0" role="region" aria-label="Chart. Highcharts interactive chart.">
<div id="highcharts-ID" class="highcharts-container " aria-hidden="false">
```

#### Live demo with steps to reproduce
Applicable to all highcharts!

#### Product version
Latest

#### Affected browser(s)
All

**Comment by goransle** [comment]
Than you for letting us know about this! We will have our accessibility specialists look further into this issue.

**Comment by wegnertm** [comment]
@goransle any updates on this? I'm running into this as well as not having a `role="img"` on the `<svg>`

**Comment by oysteinmoseng** [comment]
Hi @wegnertm!

This will be handled by the new a11y module that is coming out next year, and will no longer be an issue then.

For now, this type of issue is typically reported by automatic checkers, but should not cause any issues for users as the focus is automatically handled by the component. If however you do run into focus issues or assistive tech not handling the charts correctly, please let us know.

**Comment by nileshlad09** [comment]
Hi @oysteinmoseng,
any update on this issue?
I’m also experiencing this issue.  
The ARC Toolkit chrome extension reports the same accessibility error: “Non-active element in tab order” for elements like 
`<div id="highcharts-ID" class="highcharts-container " aria-hidden="false" tabindex="0">`
`<div id="highcharts-end-of-chart-marker-0" class="highcharts-exit-anchor" tabindex="0" aria-hidden="false"></div>`


<img width="1893" height="903" alt="Image" src="https://github.com/user-attachments/assets/4d7c61bd-f3d7-433f-aa75-9c4871d51e51" />

**Comment by oysteinmoseng** [comment]
Hi @nileshlad09 - our new a11y module is still aiming to be released this year, which would handle this differently. In the meantime you can consider these false positives by your testing tools, since focus management should be correctly handled by Highcharts in these cases.
